oauth2-server/src/main/java/com/clouway/oauth2/jwt/JwtController.java

```diff
@@ -2,10 +2,7 @@
 
 import com.clouway.friendlyserve.Request;
 import com.clouway.friendlyserve.Response;
-import com.clouway.oauth2.BearerTokenResponse;
-import com.clouway.oauth2.DateTime;
-import com.clouway.oauth2.InstantaneousRequest;
-import com.clouway.oauth2.OAuthError;
+import com.clouway.oauth2.*;
```

Comment:
You have to fix your IDEA settings in the Code Style section: 

<img width="783" alt="screen shot 2017-03-30 at 10 10 25 am" src="https://cloud.githubusercontent.com/assets/209223/24491722/2e5c9ccc-1531-11e7-8cae-cfab734ba78b.png">

Make "Class count to use *" 12 
Make "Name count to use static imports with *" 12